packages/addons/browser/chrome/source/default.py

```diff
@@ -74,28 +87,30 @@ def getAudioDevice():
                    })))['result']['value']
 
 def getAudioDeviceType(dev):
-  if dev.startswith("ALSA:"):
+  if dev.startswith("ALSA:") or dev.startswith('plughw:'):
```

Comment:
That will only solve your use case. For maximum backward compatibility the algorithm should be: if dev is not empty and does not start with "ALSA:" or "PULSE:" it is ALSA.

Reply:
@mglae I'm not really sure how to interpret the logic in your expression. Have a look whether my amended (thanks @chewitt !) PR may be more suitable.